README.md

```diff
@@ -143,6 +144,15 @@ type document
 
 ```
 
+## Community Parsers
+
+| Repo                                                                   | License                                                                            | Maintainers                                                                           | Language   | Schema v1.0 | Schema v1.1 | Links                                                                                                                                                                                                                 |
```

Comment:
Instead of `Links`, can we call it download repo?  This makes it more obvious what that is for.  Otherwise, for https://github.com/iammathew/openfga-rs, the link should also be https://github.com/iammathew/openfga-rs

Reply:
I don't think "download repo" is right. Maybe "Other Links"?

Reply:
Other/Related links should be ok.

Reply:
Updated